src/main/webapp/js/stats-loader.js

```diff
@@ -0,0 +1,23 @@
+function fetchStats(){
```

Comment:
Maybe add some documentation to this function?

Reply:
Added!